database/index.js

```diff
@@ -56,10 +56,69 @@ function getOneHomeById(id, callback) {
   Home.findById(id, callback);
 }
 
+const createHome = (body, callback) => {
+  const houseStats = { ...body };
+
+  // add "server" stuff
+  houseStats.rating = sampleData.stars[getRandomId(0, 1)];
+  houseStats.reviews = faker.random.number({ min: 20, max: 50 });
+
+  const house = new Home(houseStats);
+
+  house.save((err, home) => {
+    if (err) {
+      console.log(err);
+      callback(err);
+    } else {
+      callback(null, home);
+    }
+  });
+};
+
+const updateHome = (id, body, callback) => {
+  Home.findOne({ _id: id }, (err, doc) => {
```

Comment:
Just wondering, why don't you use **updateOne()**?

Reply:
@AndrewMyint this is a good point, I was struggling to validate whether a doc existed or not with `updateOne` but I see now I can return "number of docs updated" so that may save me a query. Thanks!

Reply:
@AndrewMyint I did a speedtest and actually this method is faster than `updateOne` so I'm keeping it